lang/es.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  lang: 'es',
+  words: [
+    {
+      // Please do not delete this one 🙏
+      name: 'spooky',
+      emoji: '761602615326146590',
+    },
+    {
+      name: 'dulce',
+      emoji: '🍬',
+    },
+    {
+      name: 'caramelo',
+      emoji: '🍬',
+    },
+    {
+      name: 'fantasma',
+      emoji: '👻',
+    },
+    {
+      name: 'halloween',
+      emoji: '🎃',
+    },
+    {
+      name: 'noche de brujas',
```

Comment:
Sorry, at the moment multi-part words don't work (see issue #9)

Reply:
Fixed